easybuild/easyconfigs/g/GULP/gulp-6.0-fftw-blas-linking-gcc-9.3.patch

```diff
@@ -0,0 +1,52 @@
+Removing hard-coded paths in the mkgulp file, which is creating the makefile
+author: J. Sassmannshausen (NHS/GSTT)
+diff --git a/gulp-6.0.org/Src/mkgulp b/gulp-6.0/Src/mkgulp
+index 28afb80..84549b0 100755
+--- a/gulp-6.0.org/Src/mkgulp
++++ b/gulp-6.0/Src/mkgulp
+@@ -170,7 +170,7 @@ fi
+ #  Include file directories
+ #
+ #--USER--Start
+-echo 'INC=-I.. -I/usr/local/include/' >> makefile
++echo 'INC=-I.. -I${EBROOTFFTW}/include -I/usr/local/include/' >> makefile
+ #--USER--End
+ #  If using Macports then use the following instead
+ #--USER--Start
+@@ -202,7 +202,7 @@ case $compiler in
+                echo 'SLIBS= ' >> makefile
+ #  If there are optimised maths libraries available replace the routines below
+ #--USER--Start
+-               echo 'MLIBS=lapack.o blas.o' >> makefile
++               echo 'MLIBS=-L${EBROOTOPENBLAS}/lib -lopenblas' >> makefile
+ #--USER--End
+            else
+ # NB: For gfortran v10 then uncomment these lines and comment the ones immediately below!
+@@ -213,8 +213,8 @@ case $compiler in
+ #  Provide the location and name of Scalapack, Blacs, Pblas, Lapack and Blas libraries.
+ #  On the Mac the following is sufficient when using Macports:
+ #--USER--Start
+-               echo 'SLIBS=-L/opt/local/lib -lscalapack ' >> makefile
+-               echo 'MLIBS= ' >> makefile
++               echo 'SLIBS=-L${EBROOTSCALAPACK}/lib -L${EBROOTOPENBLAS}/lib -lscalapack -lopenblas' >> makefile
++               echo 'MLIBS= ' >> makefile
+ #--USER--End
+            fi
+            ;;
+@@ -321,12 +321,13 @@ then
+ else
+   echo 'FDEFS=-DFFTW3 ' >> makefile
+   if [ "$parallel" = "" ]
++  then
+ #--USER--Start
+-    echo 'FLIBS=-L/usr/local/lib/ -lfftw3.a' >> makefile
++    echo 'FLIBS=-L${EBROOTFFTW}/lib -lfftw3' >> makefile
+ #--USER--End
+-  then
++  else
+ #--USER--Start
+-    echo 'FLIBS=-L/usr/local/lib/ -lfftw3 -lfftw3_mpi' >> makefile
++    echo 'FLIBS=-L${EBROOTFFTW}/lib -lfftw3 -lfftw3_mpi' >> makefile
```

Comment:
`-L${FFTW_LIB_DIR} ${LIBFFT}` but.. what of MPI? I mean you are building against a serial FFTW?